### PR TITLE
- updated adapter with attachments support

### PR DESCRIPTION
**Pull request by apugach**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -674,3 +674,97 @@ function sanitizeCollectionName (collectionName) {
...
+ * @param  {String}     collectionName  Name of the model
+ * @param  {String|Int} docId           Target document identifier
+ * @param  {String}     attName         Name of the attachment
+ * @param  {Buffer}     attBuffer       The buffer that represents attachment content
+ * @param  {String}     attType         The MIME type of the attachment
+ * @return {Promise}                    Resolves attahcment operation result when succeded
+ */
+adapter.attach = function create(connectionName, collectionName, docId, attName, attBuffer, attType) {
+  return new Promise(function (fulfill, reject) {
+    try {
```
Comment (screamy) [inline index.js]: you don't need to wrap code inside promise in try/catch clause

File: index.js [hunk 2]
```diff
@@ -674,3 +674,97 @@ function sanitizeCollectionName (collectionName) {
...
+ *
+ * Destroys an attachment in the specified document
+ *
+ * @param  {String}     connectionName  Name of the connection
+ * @param  {String}     collectionName  Name of the model
+ * @param  {String|Int} docId           Target document identifier
+ * @param  {String}     attName         Name of the attachment
+ * @return {Promise}                    Resolves attahcment destroyed
+ */
+adapter.deattach = function create(connectionName, collectionName, docId, attName) {
```
Comment (screamy) [inline index.js]: probably `detach` looks better

File: index.js (hunk 1 above)
Comment (apugach) [inline index.js]: fixed

File: index.js (hunk 2 above)
Comment (apugach) [inline index.js]: renamed to `detach`

**Pull request merged**